source/user-guide/creating-environments.html.md.erb

```diff
@@ -66,6 +66,16 @@ The idea of sandbox access is that it gives you freedom to experiment and modify
 
 Anything not in code will be lost at the end of the week.
 
+#### migration
+
+The migration role is a temporary role, given to assist with AWS managed migration services
+
+- Log into the console
+- Developer permissions
+- Permissions to use [AWS Migration services](https://docs.aws.amazon.com/whitepapers/latest/aws-overview/migration-services.html)
+
+Once the migration of an application is complete this role should be removed.  No in production applications should have this role.
```

Comment:
Should this guidance slightly change to something akin to "this role should be removed once used for the migration"? It currently reads as `production` _environments_ should not have this role - but if you're migrating a production database, you probably wouldn't want to copy it to a lower environment first?